ztp/ztp-policy-generator/Makefile

```diff
@@ -0,0 +1,33 @@
+PWD := $(shell pwd)
+BASE_DIR := $(shell basename $(PWD))
+
+# Keep an existing GOPATH, make a private one if it is undefined
+GOPATH_DEFAULT := $(PWD)/.go
+export GOPATH ?= $(GOPATH_DEFAULT)
+GOBIN_DEFAULT := $(GOPATH)/bin
+export GOBIN ?= $(GOBIN_DEFAULT)
+export PATH := $(PATH):$(GOBIN)
+XDG_CONFIG_HOME=./
+KUSTOMIZE_PLUGIN_HOME ?= $(XDG_CONFIG_HOME)/kustomize/plugin
+
+.PHONY: build test test-kustomize all clean
+build:
+	go build -mod=vendor -o policygenerator/PolicyGenerator ./policygenerator/
+
+test:
+	go test ./policygenerator/...
+
+test-kustomize: build
+	cp policygenerator/PolicyGenerator kustomize/plugin/policyGenerator/v1/policygenerator/PolicyGenerator
+	cp -r policygenerator/resources kustomize/plugin/policyGenerator/v1/policygenerator/
+	kustomize build --enable-alpha-plugins
```

Comment:
kustomize isn't installed as part of the openshift-ci image that GitHub uses for testing, so this will probably fail.